catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -595,7 +595,6 @@
             persistent-id="org.codice.ddf.catalog.ui.query.monitor.impl.WorkspaceQueryService"
             update-strategy="container-managed"/>
         <property name="queryTimeoutMinutes" value="5"/>
-        <property name="cronString" value="0 0 0 * * ?"/>
```

Comment:
If you are deleting this property, can this method be deleted. https://github.com/codice/ddf/blob/master/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceQueryServiceImpl.java#L172

Reply:
I spoke to @millerw8 about that and he said to remove the cron string config but leave the logic in for other programs